manim/animation/updaters/mobject_update_utils.py

```diff
@@ -99,7 +99,39 @@ def construct(self):
     return mob
 
 
-def always_shift(mobject, direction=RIGHT, rate=0.1):
+def always_shift(
+    mobject: Mobject, direction: np.ndarray[np.float64] = RIGHT, rate: float = 0.1
+) -> Mobject:
+    """Shift the mobject by `rate*direction` every second
+
+    Parameters
+    ----------
+    mobject
+        The mobject to shift
+    direction
+        The direction to shift
+    rate
+        Amount of direction to be covered in one second
+
+    Examples
+    --------
+
+    .. manim:: ShiftingSquare
+
+        class ShiftingSquare(Scene):
+            def construct(self):
+                sq = Square().set_fill(opacity=1)
+                tri = Triangle()
+                VGroup(sq, tri).arrange(LEFT)
+
+                # always shift square one unit to the right
+                # even if there is an animation going on
+                always_shift(sq, RIGHT, rate=5)
+
+                self.add(sq)
+                self.play(tri.animate.set_fill(opacity=1))
+    """
+
     def normalize(v):
```

Comment:
This should be replaced by a call to `normalize` from `manim.utils.space_ops`.

Reply:
done!